### ISSUE TITLE
Ugly scrollbar at the sidebar

### ISSUE THREAD

**Issue by AndreasHohnholt** [creativetimofficial/light-bootstrap-dashboard-pro-angular#17]
at the right side of the sidebar is a ugly scrollbar, so that there is a big gap between the sidebar and the navbar.  the non angular version has not this issue.

Angular version: https://lbd-pro-angular2.creative-tim.com/dashboard
Non Angular vesion: https://demos.creative-tim.com/light-bootstrap-dashboard-pro/examples/dashboard.html

**Comment by chelaruc** [comment]
@AndreasHohnholt Can you tell me what OS do you use and what browser?

**Comment by AndreasHohnholt** [comment]
The problem occur in all major browser:

- Google Chrome Version 64.0.3282.186
- Microsoft Edge 40.15063.674.0
- Firefox 53.0.2 (32-Bit) / 56.0 (32-Bit)
OS: Windows 10 Pro 64 Bit 

- Safari 11.0.3 on MacOS


[Screenshot](https://ibb.co/jqO3vS)

**Comment by gusjabar** [comment]
I have the same problem, I guess it's something related with the library perfect-scrollbar.js.

**Comment by AndreasHohnholt** [comment]
on my samsung galaxy tab s3 the error does not occur, no matter which browser is used. if i emulate an "ipad pro" in the chrome browser the error does not occur either.

**Comment by chelaruc** [comment]
Hi @AndreasHohnholt! Can you delete the line 20 from `src/index.html` and in `src/assets/sass/lbd/_sidebar-and-main-panel.scss` delete line 12: `overflow-y: scroll;`

Tell me if this worked.

**Comment by AndreasHohnholt** [comment]
@chelaruc  Yes ist fix the problem but there is now a small unattractive corner at the end of the scrollbar https://ibb.co/ksYjs7

**Comment by chelaruc** [comment]
@AndreasHohnholt in `src/assets/sass/lbd/_sidebar-and-main-panel.scss`, on class .sidebar-wrapper you need to change the height from `height: calc(100% - 75px);` to `height: calc(100% - 64px);`

**Comment by AndreasHohnholt** [comment]
Will this issue fixed in the next release of light-bootstrap-dashboard-pro-angular?

**Comment by chelaruc** [comment]
@AndreasHohnholt yes, we will fix this in next update. I'll reopen this issue and close when the update will be released.
